# Winerror-10049-in-jupyter-notebook

(base) C:\Users\user>jupyter notebook
[W 17:28:01.211 NotebookApp] Cannot bind to localhost, using 127.0.0.1 as defaul
t ip
    [WinError 10049] The requested address is not valid in its context
[I 17:28:04.608 NotebookApp] Serving notebooks from local directory: C:\Users\us
er
[I 17:28:04.608 NotebookApp] The Jupyter Notebook is running at:
[I 17:28:04.609 NotebookApp] http://127.0.0.1:8888/
[I 17:28:04.609 NotebookApp] Use Control-C to stop this server and shut down all
 kernels (twice to skip confirmation).
[I 17:28:22.281 NotebookApp] Creating new notebook in /Documents/Python Scripts
